build/Uno.UI.Build.csproj

```diff
@@ -42,10 +42,6 @@
 		<CallTarget Targets="PrepareNuGetPackage" />
 	</Target>
 
-	<Target Name="GeneratePackages" AfterTargets="Build" DependsOnTargets="PrepareBuildAssets" Condition="'$(BuildingInsideVisualStudio)'==''">
-		<CallTarget Targets="BuildNuGetPackage" Condition="'$(Configuration)'=='Release'" />
-	</Target>
```

Comment:
This target is only a wrapper around `BuildNuGetPackage`, so I kept only `BuildNuGetPackage`